site/styles/_availability-tags.scss

```diff
@@ -1,43 +1,63 @@
-s72-availability-status,
 s72-availability-label {
-  position: absolute;
-  z-index: 99;
-}
-
-.availability-bg {
-  background: rgba(0, 0, 0, 0.8);
-  display: block;
-  padding: 5px 8px;
-  text-align: center;
-
+  font-weight: $font-weight-light;
   .availability-state {
-    font-size: 12px;
-    font-weight: $font-weight-bold;
-    text-transform: none;
-  }
-}
+    @extend .d-flex;
+    line-height: 24px;
+    @extend .mb-2;
+    font-size: 16px;
+    letter-spacing: 0.4px;
 
-.availability-bg,
-.availability-state {
-  font-size: 12px;
-}
-
-.availability-state {
-  color: #fff;
-  font-weight: $font-weight-bold;
-  text-transform: uppercase;
+    &:before {
+      @extend .fa;
+      @extend .fa-exclamation-circle;
+      @extend .mr-1;
+      color: var(--body-color);
+      line-height: 24px;
+      font-size: 24px;
+    }
+  }
 }
 
 .poster {
+  s72-availability-status,
   s72-availability-label {
-    left: 0 !important;
-    margin-left: auto;
-    margin-right: auto;
-    padding: 10px 10px 5px;
-    right: 0;
+    position: absolute;
+    z-index: 99;
+  }
+
+  s72-availability-label {
+    background-color: rgba(var(--body-bg-rgb),0.7);
```

Comment:
Shouldn't this be on the `.availability-bg` element?

Reply:
Also... your comment said you put it back to 0.8 opacity... was that a typo?

Reply:
- Ah yes .availability-bg makes more sense. Changed.
- Yeah 0.8 opacity was a typo. I think i set it to 0.8, then realised 0.7 was a good medium when the design wanted 0.5